rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -544,13 +546,27 @@ public ResponseEntity<TaskRepresentationModel> terminateTask(@PathVariable Strin
   public ResponseEntity<TaskRepresentationModel> transferTask(
       @PathVariable String taskId,
       @PathVariable String workbasketId,
-      @RequestBody(required = false) Boolean setTransferFlag)
+      @RequestBody(required = false)
+          TransferTaskRepresentationModel transferTaskRepresentationModel)
```

Comment:
What do you think about renaming this class? For me, the name is not intuitive. I would still be ok with leaving it like this

Reply:
How do you want to rename it?

Reply:
We should discuss it :)